helper/helper.go

```diff
@@ -0,0 +1,81 @@
+package helper
+
+import (
+	"archive/tar"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func GetApplication(applicationsPath, applicationUUID string) (string, string, error) {
```

Comment:
Note you can specify return values in the signature in go, e.g.:

```go
func GetApplication(applicationsPath, applicationUUID string) (app string, commit string, err error) {
    // assign the vals...
    // ...
    return
}
```

The unconditional `return` will return the values according to what they are set to. They will default to their 'zero values', i.e. 0 for an integer, nil for a pointer etc.